### ISSUE TITLE
[question] use colmap2nerf.py to extract 94 imgs on my own data. Only 2 imgs are logged in transforms.json. Why?

### ISSUE THREAD

**Issue by zydmtaichi** [NVlabs/instant-ngp#1554]
I run script to convert my video to nerf dataset. the program extract 94 imgs but only 2 of these are written to transforms.json. pls explain what problem cause this?

**Comment by casualdays** [comment]
Maybe you should improve the video (or image) quality you shoot (use 4k 60FPS), move slowly around the object when shooting, and preferably use an anti-shake device.

**Comment by TuXun1999** [comment]
I had a similar experience. My solution was to change the colmap camera model. By default, it's OPENCV. I changed it into PINHOLE, and successfully extracted out about 150 images from the video. To do this, check colmap2nerf.py

**Comment by zydmtaichi** [comment]
> I had a similar experience. My solution was to change the colmap camera model. By default, it's OPENCV. I changed it into PINHOLE, and successfully extracted out about 150 images from the video. To do this, check colmap2nerf.py

yes, I get 92 extracted logs of 94 imgs in PINHOLE mode while only 2 extracted under OPENCV mode
